server/api/batch_inventory.py

```diff
@@ -663,11 +663,6 @@ def download_batch_inventory_ballot_manifest(
     batch_inventory_data = get_or_404(BatchInventoryData, jurisdiction.id)
     election_results: ElectionResults = batch_inventory_data.election_results
 
-    if not batch_inventory_data.signed_off_at:
```

Comment:
I could be smart and only perform this check if showing the ballot manifest file (and by extension including the sign-off step). But that configuration isn't stored on the backend right now and just lives in frontend JS (as you saw in https://github.com/votingworks/arlo/pull/1923/commits/1c3aec29ee9d7fdab815a35a364546dca0a4b261). I could have the frontend pass that config up to the backend, but that didn't feel worth the trouble. Long-term, we'll almost certainly be moving the config to the backend